src/main/java/org/sonarsource/sonarlint/ls/SonarLintExtendedLanguageServer.java

```diff
@@ -350,6 +351,15 @@ public void setFileUri(String fileUri) {
   @JsonNotification("sonarlint/openHotspotInBrowser")
   void openHotspotInBrowser(OpenHotspotInBrowserLsParams params);
 
+  @JsonNotification("sonarlint/openDependencyRiskInBrowser")
+  CompletableFuture<Void> openDependencyRiskInBrowser(OpenDependencyRiskInBrowserParams params);
+
+  record OpenDependencyRiskInBrowserParams(UUID issueId, String folderUri) {
+  }
+
+  @JsonNotification("sonarlint/dependencyRiskIssueInvestigatedLocally")
+  CompletableFuture<Void> dependencyRiskIssueInvestigatedLocally();
```

Comment:
Keeping this as `CompletableFuture<Void>` intentionally so that I can cover it with tests